### DISCUSSION TITLE
Linking between drafts and meetings

### DISCUSSION THREAD

**Discussion by mnot** [ietf-tools/datatracker#6360] [Ideas]
It would be really nice if there were an easy way to see which meetings a particular draft was discussed in, with a link to the approppriate minutes.

E.g., when I look at draft-nottingham-foo-bar, I should be able to follow a link to the FICTION WG session in IETFnnn when it was discussed. If multiple sessions were relevant (across one or more meetings), each should be linked.

**Comment by rjsparks** [comment]
This capability (almost) already exists. It _did_ exist at one point in the past, with a list of sessions on the document's main page, but people found that too noisy. You can see it now (but without links) on the history tab of a document - look for "Added to session" in https://datatracker.ietf.org/doc/draft-ietf-core-href/history/ for example.

It requires chairs telling the datatracker which drafts are relevant to a meeting. There is an Internet-Drafts section of the meeting materials page where I-Ds can be added or removed for each session. For any session in the _future_, the meeting will show on the document's main page right now - it's the past sessions that were taken away. We could look into providing a less intrusive bit of UI that would let you get to them.

Note that this does not do anything like attempt to heuristically scrape agendas. Chairs have to be explicit in attaching a draft to a session.

**Comment by mnot** [comment]
Considering the amount of information already on a document's main page (most of which I never use), I find it a bit surprising/ironic that people could feel that this extremely relevant information is "noisy". To each their own, I suppose.

Perhaps a suitable UI would be an extra tab, alongside 'history' -- something like 'discussions'? There you could link to each meeting the document was discussed in. Perhaps even allowing someone to give a direct link to the appropriate portion of the minutes?

If it were possible to use an API to associate a meeting with a document -- ideally for past meetings as well as current ones -- that would be most useful; writing a local tool to do the scraping would be trivial.

It strikes me that this _could_ be hacked in already using `Additional Resources` -- but I suspect that might make some unhappy :)

**Comment by Charlotte-br560** [comment]
Absolutely! This feature would greatly enhance traceability and streamline the process. Integrating meeting references directly with drafts can save a lot of time and effort for everyone involved. Plus, it ensures nothing gets lost in the shuffle.
